skins/kdark/css/layout.css

```diff
@@ -1972,6 +1972,11 @@ background-image:  linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transpa
     position: absolute;
     -webkit-tap-highlight-color: rgba(0,0,0,0);
 }
+
+#flagComments.validationError{
+	outline: none !important;
```

Comment:
Is the `!important` required? it seems to set outline to 0 even without the important.